### ISSUE TITLE
Stop decoding of a file

### ISSUE THREAD

**Issue by karsten42** [qedus/osmpbf#25]
Is there a way right now to stop the decoding of a file? 
As far as I see from looking at the code if you were to stop calling the `Decode()` method before you went through the entire file there would be dangling go routines that you can't stop. Am I missing something, or is it just not intended so far to stop the decoding prematurely?

**Comment by AlekSi** [comment]
You are right, it's not implemented yet. Feel free to send a PR if you have time.
